### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ghost**
updating for consistency

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,20 +5,20 @@
 [build-status-image]: https://travis-ci.org/kaazing/ios.client.svg?branch=develop
 [build-status]: https://travis-ci.org/kaazing/ios.client
 
-# About this Project
+# About This Project
 
 ios.client is a parent project of iOS based client projects.
```
Comment (vjwang) [inline README.md]: 1. Project-level READMEs would be helpful, even if they briefly describe what they are and point to the parent README.
2. The About section should explain the repo and what's included in the projects. See the [javascript.client About section](https://github.com/kaazing/javascript.client/blob/develop/README.md), for example.

Comment (ghost) [inline README.md]: 1. I can do top-level READMEs as part of this story. I added a task to the story.
2. I think the About section can contain a single sentence or multiple sentences. It's at the discretion of the dev who writes the content. I'm just ensuring that the section has something.

Please accept this pull request.

File: README.md [hunk 2]
```diff
@@ -26,17 +26,20 @@ cd appledoc
...
+## Steps for Building This Project
 
 0. Clone the repo: ```git clone https://github.com/kaazing/ios.client.git```
 0. Go to the cloned directory: ```cd ios.client```
 0. Build the project: ```mvn clean install```
 
+# Learning How to Use the Gateway
+
+To learn how to administer the Gateway, its configuration files, and security, see the documentation on [developer.kaazing.com](http://developer.kaazing.com/documentation/5.0/index.html).
+
```
Comment (vjwang) [inline README.md]: Since this is the CE client, should this section also refer to a generic RFC 6455 endpoint?

Comment (ghost) [inline README.md]: I don't think so. I'm just telling them about the Gateway here. There's no other RFC 6455 endpoint to discuss.

File: README.md [hunk 3]
```diff
@@ -1,42 +1,5 @@
...
-0. Build the project: ```mvn clean install```
-
-# Learning How to Develop Client Applications
-
-Learn to develop RFC-6455 based [iOS client applications](http://kaazing.org/documentaton/5.0/dev-ios/o_dev_ios.html).
-
-# View a Running Demo
-
-View a demo (see kaazing.org)
+The ios.client project includes the following projects:
```
Comment (vjwang) [inline README.md]: "project"

**Pull request merged**